lib/vsc/install/shared_setup.py

```diff
@@ -800,8 +801,9 @@ def loadTestsFromModule(self, module, pattern=None):  # pylint: disable=argument
                 try:
                     # pattern is new, this can fail on some old setuptools
                     testsuites = ScanningLoader.loadTestsFromModule(self, module, pattern)
-                except TypeError:
+                except TypeError as e:
                     log.warn('pattern argument not supported on this setuptools yet, ignoring')
+                    log.warn(f'original exception {e} {traceback.format_exc()}')
```

Comment:
can't you use log.exception() to print the traceback?

Reply:
the `log` here is not `logging` related

Reply:
that is indeed for next gen vsc-install without fancylogger.